### ISSUE TITLE
Block controls occupied by mod's controls

### ISSUE THREAD

**Issue by E66666666** [E66666666/GTAVManualTransmission#13]
When using a controller, controls can be bound to buttons that the game also uses for existing actions. It'd be nice if this could be blocked properly without losing original functionality.

PR #12, and commits c6cf9a9 and 58b6a2e solved the tapping thing, but blocking things is still lacking.

Any ideas?

**Comment by E66666666** [comment]
26c3380e62c7dd83d1e2e8cfa3bc9f80074fb13b enabled executing the original functions after holding the buttons, but I can't get single-tap actions to work. Example:

Shift Up blocks the Duck action. Tapping Shift Up just makes the mod shift up, and the Duck action is blocked properly. Holding Shift Up executes the Duck action and the character can be seen ducking while this button is held. The mod doesn't shift up.

Shift Down blocks the Switch Weapon action. Tapping Shift Down just makes the mod shift down, and the Switch Weapon action is blocked properly. Holding Shift Down **does _not_** execute the _Switch Weapon_ action. The mod doesn't shift down.

Help would be nice.

**Comment by peaches6** [comment]
Have you found a solution to this? If I understand correctly, holding either `shift up` or `shift down` does not call the respective function, but instead in `shift up`'s case, causes the character to duck, and in `shift up`'s case calls nothing.

**Comment by E66666666** [comment]
Nope. Depending on the original action, holding works as long as the original action also allows holding. This is ducking, aiming, shooting, etc. If the original action was tap-only (switch weapons), it doesn't work.

**Comment by E66666666** [comment]
Is is still open, by the way.

**Comment by Luc45** [comment]
Howdy!

"X" is mapped to `downshift`by default, but it is also mapped to the `Next vehicle weapon` action natively. Is there a way to prevent "X" from triggering the `Next vehicle weapon` action?

![image](https://user-images.githubusercontent.com/9341686/99906191-772b6480-2cb4-11eb-9894-ceaca57d93d5.png)
